l5kit/l5kit/rasterization/satellite_rasterizer.py

```diff
@@ -51,35 +48,30 @@ def rasterize(
     ) -> np.ndarray:
 
         if agent is None:
-            ego_translation = history_frames[0]["ego_translation"]
+            ego_translation_m = history_frames[0]["ego_translation"]
             # Note 2: it looks like we are assuming that yaw in ecef == yaw in sat image
```

Comment:
Somewhat confusing note, are we not sure?

Reply:
I'm not sure of anything in that rasterizer 😄

Reply:
We can have another round of refactoring in qualify week